dashboard/src/main/java/org/openedx/learn/presentation/LearnViewModel.kt

```diff
@@ -43,6 +49,12 @@ class LearnViewModel(
 
     init {
         viewModelScope.launch {
+            pushNotifier.notifier.collect { event ->
```

Comment:
Everything is good here. I just have a suggestion to update the order of these methods.

- The 'courseNotifier` should be first.
- The `updateLearnType` should be above the two notifications-related function.

Reply:
addressed.